config/initializers/huginn_agent.rb

```diff
@@ -0,0 +1,5 @@
+require 'huginn_agent'
+
+HuginnAgent.hack_huginn_to_accept_me
```

Comment:
We could certainly modify `AssignableTypes` to make it easier to add types after the fact.  Wouldn't it be as simple as this?

```ruby
    def add_type(type)
      const_get(:TYPES).push(type)
    end

   ...

   Agent.add_type "Agents::NewAgent"
```